src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantsStatsLoad.java

```diff
@@ -61,7 +61,7 @@ public RepeatStatus execute(StepContribution contribution, ChunkContext chunkCon
         QueryOptions statsOptions = new QueryOptions(variantOptions);
         VariantStatisticsManager variantStatisticsManager = new VariantStatisticsManager();
         VariantDBAdaptor dbAdaptor = variantStorageManager.getDBAdaptor(variantOptions.getString("dbName"), variantOptions);
```

Comment:
`variantOptions.getString(VariantStorageManager.DB_NAME)` could be used, same as in `VariantStatsCreate`.